src/math_parser_diag.cpp

```diff
@@ -627,20 +627,19 @@ std::function<double( dialogue & )> proficiency_eval( char scope,
         time_duration raw = d.actor( beta )->proficiency_practiced_time( prof );
         std::string const format = fmt_val.str( d );
         if( format == "percent" ) {
-            return static_cast<int>( raw * 100  / prof->time_to_learn() );
+            return static_cast<double>( raw * 100  / prof->time_to_learn() );
```

Comment:
```suggestion
            return raw * 100.0  / prof->time_to_learn();
```
same for `"permille"`